src/Makefile

```diff
@@ -136,7 +136,7 @@ FLAGS += -DLLVM_SHLIB
 endif # USE_LLVM_SHLIB == 1
 endif
 
-RT_LLVM_LINK_ARGS := $(shell $(LLVM_CONFIG_HOST) --libs $(RT_LLVM_LIBS) --system-libs --link-static)
+RT_LLVM_LINK_ARGS := $(shell $(LLVM_CONFIG_HOST) --libs $(RT_LLVM_LIBS) -lrt -ldl -lpthread -lz --link-static)
```

Comment:
these are arguments to `llvm-config`. You want to add them to `RT_LLVMLINK`, but they are also OS specific, so see below for how to do that.